README.md

```diff
@@ -1,14 +1,73 @@
 # Juttle Client Library
 
+Juttle Client Library is a set of lower level frontend building blocks for
```

Comment:
`lower level` is relative, but not clear to what. Suggest removing and saying `set of frontend...`.